test/jdk/sun/tools/jhsdb/JShellHeapDumpTest.java

```diff
@@ -53,7 +53,7 @@ public class JShellHeapDumpTest {
     static Process jShellProcess;
     static boolean doSleep = true; // By default do a short sleep when app starts up
 
-    public static void launch(String expectedMessage, List<String> toolArgs)
+    public static boolean launch(String expectedMessage, List<String> toolArgs, boolean allowRetry)
```

Comment:
You should add a comment explaining the return value like we have for printStackTraces().